### ISSUE TITLE
Expose version retrieval timestamp on HTTP API

### ISSUE THREAD

**Issue by gasinvein** [fedora-infra/anitya#1082]
The "**Retrieved on (UTC)**" field could be useful for HTTP API users, who may want to know when the version was released without further crawling the upstream project, if Anytia provides this information.

**Comment by Zlopez** [comment]
It should be easy to add this field to API.

**Comment by gasinvein** [comment]
`versions` and `stable_version` are plain string lists - I guess timestamps can't be added there, so we'll need a new field to contain list of `(version, timestamp)` pairs?

**Comment by Zlopez** [comment]
You are right, this would be probably a backwards incompatible change, because you need to change the type to object from list.
